src/handlers/monthly-post.js

```diff
@@ -20,9 +76,9 @@ function formatPostText(options) {
     `スパイク禁止のため、ランシューズまたはトレーニングシューズを用意してください。`,
     `雨天で中止になる可能性もあります。当日にまた投稿するので確認ください。`,
     ``,
-    `Weeknight Hiroo Pickup dates for April/May`,
-    `April 6 (Wed), 13 (Wed), 20 (Wed), 27 (Wed)`,
-    `May 4 (Wed), 11 (Wed), 18 (Wed), 25 (Wed)`,
+    `Weeknight Hiroo Pickup dates for ${options.monthNames.en.join('/')} `,
+    `${options.datesListByMonth.en[0]}`,
+    `${options.datesListByMonth.en[1]}`,
```

Comment:
I wonder if we should just make it a structure that explicitly has month one and month two, or whether this should iterate through an array. Not necessary now but useful for the more general case I guess.

Reply:
There's actually a bit wonkiness here that I don't like, and it's that the position of the language property isn't consistent. For `monthNames` it's the most specific (last-defined) property, but for datesListByMonth it's actually second-to-last:
```
datesListByMonth = {
    en: [
        "April 6 (Wed), 13 (Wed), 20 (Wed), 27 (Wed)",
        "May 4 (Wed), 11 (Wed), 18 (Wed), 25 (Wed)"
    ],
    jp: [
        // etc
    ]
}
```

I think we can revisit this when we think about what "variables" to provide in the case we open this up for other pickups. For now we can easily rely on Hiroo being strictly 2 months at a time.